src/hooks/useFeatureCards.ts

```diff
@@ -18,6 +18,7 @@ export const useFeatureCards = (): UseFeatureCardsProps => {
   apiUrl.searchParams.set('populate[0]', 'BackgroundImageLight');
   apiUrl.searchParams.set('populate[1]', 'BackgroundImageDark');
   import.meta.env.MODE !== 'production' &&
+    import.meta.env.MODE !== 'staging' &&
```

Comment:
Super small change but don't we have a 'development' MODE?

Reply:
YES! That simplifies the condition a lot :D Thanks!